ext/version_sorter/version_sorter.c

```diff
@@ -232,11 +240,25 @@ rb_version_sort_r_bang(VALUE rb_self, VALUE rb_versions)
 	return rb_version_sort_1(rb_self, rb_versions, version_compare_cb_r);
 }
 
+static VALUE
+rb_version_compare(VALUE rb_self, VALUE rb_version_a, VALUE rb_version_b)
+{
+	return INT2NUM(rb_version_compare_1(rb_self, rb_version_a, rb_version_b, version_compare_cb));
+}
+
+static VALUE
+rb_version_compare_r(VALUE rb_self, VALUE rb_version_a, VALUE rb_version_b)
```

Comment:
Since we're exposing a low-level function, I don't think it makes sense to expose an alternate `rcompare()` function that simply returns a negated result of `compare()`. If the user needs to write low-level comparison code, then they can negate the result of `compare()` themselves, and they don't strictly need `rcompare()`. Does that make sense?

If we got rid of `rcompare`, we wouldn't have the need for the internal `rb_version_compare_1` function either.